### PR TITLE
Add WhatsApp and AppSignal as sponsors

### PR DESCRIPTION
**Pull request by pablocostass**

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -90,6 +90,12 @@ platinum_sponsors:
   - name: Remote
     logo: sponsors_2021/remote.png
     url: https://remote.com/
+  - name: WhatsApp
+    logo: sponsors_2021/whatsapp.png
+    url: https://www.whatsapp.com/
+  - name: AppSignal
+    logo: sponsors_2021/appsignal.png
+    url: https://www.appsignal.com/elixir?utm_source=spawnfest_sponsored&utm_medium=web
```
Comment (paulo-ferraz-oliveira) [inline _config.yml:98]: I'd prefer it if this was just `appsignal.com`.

Why?
1. we support ALL BEAM languages and this is kinda like a statement that Elixir is taking preference,
2. it's AppSignal that's sponsoring, not AppSignal/Elixir,
3. we can think about this sort of thing for next year (making it more explicit in the registration form that we support BEAM, not a particular language, for example, or being open to exceptions).

Thoughts?

File: _config.yml [hunk 2]
```diff
@@ -141,6 +147,14 @@ prizes:
     sponsor: Remote
     sponsor_url: https://remote.com/
     image: prizes_2021/oculus-quest-2.png
+  - name: $1500 in swag
+    sponsor: WhatsApp
+    sponsor_url: https://www.whatsapp.com/
+    image: prize-placeholder.png
+  - name: 3x €500 in AppSignal credits
+    sponsor: AppSignal
+    sponsor_url: https://www.appsignal.com/elixir?utm_source=spawnfest_sponsored&utm_medium=web
```
Comment (paulo-ferraz-oliveira) [inline _config.yml:156]: Same comment as above, for the Elixir part.

**Comment by paulo-ferraz-oliveira** [comment]
Is the WhatsApp symbol not green?

**Comment by pablocostass** [comment]
> Is the WhatsApp symbol not green?

It is, but if you check their [branding page](http://whatsappbrand.com/) you will see that they do not have publicly available their white and green logo with a dark font. I did that logo myself at f475916 (https://github.com/spawnfest/spawnfest.github.io/blob/f475916e01e079965b967ab58b4c0ce6e68705d4/img/sponsors_2021/whatsapp.png), but I decided to not use it just in case they would not like it.

I can always revert the change if you like it though

**Comment by starbelly** [comment]
@pablocostass  The WhatsApp image makes sense to me per their guidelines.

**Review by starbelly — APPROVED** [review]

**Pull request merged**